textattack/shared/utils/misc.py

```diff
@@ -86,3 +86,4 @@ def set_seed(random_seed):
     random.seed(random_seed)
     np.random.seed(random_seed)
     torch.manual_seed(random_seed)
+    torch.cuda.manual_seed(random_seed)
```

Comment:
wow i didn't know we also had to set seed for CUDA. Good catch!

Reply:
ikr